lib/lints/avoid_late_keyword/models/avoid_late_keyword_parameters.dart

```diff
@@ -0,0 +1,24 @@
+part '_config_parser.dart';
+
+/// A data model class that represents the "avoid late keyword" input
+/// parameters.
+class AvoidLateKeywordParameters {
+  /// Allow to dynamically initialize
```

Comment:
```suggestion
  /// Allow immediately initialised late variables.
  /// 
  /// ```dart
  /// late var ok = 0; // ok when allowInitialized == true
  /// late var notOk; // initialized elsewhere, not allowed
  /// ```
```